float32 x
float32 y
float32 z_min
float32 z_max
uint8   type

uint8 FREE       = 0
uint8 LOW        = 1
uint8 HIGH       = 2
uint8 DYNAMIC    = 3
uint8 STREET     = 4
uint8 UNKNOWN    = 255
